bytes.go

```diff
@@ -25,6 +30,8 @@ var (
 	oldBytesUnitMap = MakeUnitMap("B", "B", 1024)
 )
 
+var ErrIncorrectType = errors.New("not a string or int64")
```

Comment:
can be int, int8, int16, int32 as well

Reply:
It can be only int64 or string. 

TOML spec says that all ints are `int64`:

> 64 bit (signed long) range expected (−9,223,372,036,854,775,808 to 9,223,372,036,854,775,807).

https://github.com/toml-lang/toml#user-content-integer

Reply:
may be you should rename error? it is toml specific incorrect type error, but not package-wide incorrect type error.